source/app.d

```diff
@@ -51,13 +52,22 @@ void main(string[] args)
 	if (!CLI!DlingsCmdlArgs.parseArgs(cmdlArgs, args[1 .. $]))
 		return;
 
+
+	if (!exists("exercises.toml"))
+	{
+		writeln("file 'exercises.toml' is missing!");
```

Comment:
Write errors to `stderr` instead.

Reply:
Fixed in 0bde9959dc44173e02db1b48c6fa65a40461c29a